cheatsheets/Clickjacking_Defense_Cheat_Sheet.md

```diff
@@ -2,7 +2,7 @@
 
 This cheat sheet is focused on providing developer guidance on [Clickjack/UI Redress](https://www.owasp.org/index.php/Clickjacking) attack prevention.
 
-The most popular way to defend against Clickjacking is to include some sort of "frame-breaking" functionality which prevents other web pages from framing the site you wish to defend. This cheat sheet will discuss two methods of implementing frame-breaking: first is X-Frame-Options headers (used if the browser supports the functionality); and second is javascript frame-breaking code.
+The most popular way to defend against Clickjacking is to include some sort of "frame-breaking" functionality which prevents other web pages from framing the site you wish to defend. This cheat sheet will discuss two methods of implementing frame-breaking: first is X-Frame-Options headers (used if the browser supports the functionality); and second is JavaScript frame-breaking code.
```

Comment:
- Add a link just like this [X-Frame-Options](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options) header; it is a singular header, and not multiple ones.
- and `the` second is ... Add `the`
- This introduction looks to be old, and I believe could be updated. Could you possible open up an issue about this in order to handle it? 😄

Reply:
I hadn't really look at the intro in any depth, but you're right that it didn't read too well. I've had a stab at rewriting it in the latest commit.